src/tests/ftest/util/dfuse_utils.py

```diff
@@ -269,6 +269,10 @@ def run(self, check=True):
             raise CommandFailure(
                 "Dfuse missing environment variables for D_LOG_FILE")
 
+        # self.env['DD_MASK'] = 'all'
+        # self.env['DD_SUBSYS'] = 'all'
+        self.env['D_LOG_MASK'] = 'INFO'
```

Comment:
This isn't ideal, since not all tests using dfuse would want this. There isn't a clean way to set env vars, which is something I'm working on independently. Can we at least do this so other tests aren't overridden? (also recommend removing unused)
```suggestion
        if "D_LOG_MASK" not in self.env:
            self.env['D_LOG_MASK'] = 'INFO'
```

Reply:
I've added the conditional.  I've been sent failing tests multiple times where when I've asked where the dfuse logs are we've discovered they've never been collected and we have no idea how to enable them.  I agree it's ugly but adding these env vars and running existing test is something that I do a lot so having the comment there would really help this process.  Perhaps having the D_LOG_MASK will make it easier to know where to add the others if required though.